app/assets/javascripts/IndividualResult.js

```diff
@@ -0,0 +1,46 @@
+const mongoose = require('mongoose');
+
+const [Number, String, Mixed, ObjectId] = [
+  mongoose.Schema.Types.Number,
+  mongoose.Schema.Types.String,
+  mongoose.Schema.Types.Mixed,
+  mongoose.Schema.Types.ObjectId,
+];
+
+const IndividualResultSchema = mongoose.Schema(
+  {
+    // Population Attributes
```

Comment:
Encapsulate these in an object so they can be iterated over, and so the types can be different between episode/population-based measures

Reply:
If episode-based: each population attribute would return an Array of episode results